addons/website_livechat/static/src/legacy/public_livechat.scss

```diff
@@ -1,11 +1,3 @@
-.editor_has_snippets:not(.o_fullscreen) {
-    .o_livechat_button, .o_thread_window {
-        // TODO add this in an edit-mode only file in master (in 14.0 that asset
-        // would be website.assets_wysiwyg...)
-        right: $o-we-sidebar-width !important;
-    }
-}
-
```

Comment:
Does this do nothing anymore?

I see that the TODO specifies to move it, but you removed it?
Or maybe since we now hide the buttons it's not necessary anymore?

Reply:
@awa-odoo Indeed, that is the case. Now that it is hidden in edit mode, we don't need to move it to the left of edition sidebar :)

Reply:
Might be cool mentioning in the commit message
https://www.odoo.com/web#id=2369225&cids=1&model=project.task&view_type=form
and/or https://github.com/odoo/odoo/commit/3f8e4f5eef8c1828bd599a021342398a9097fda5
One might wonder why was this existing in the first place and try to blame its origin, it will help